pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceChaosTest.java

```diff
@@ -95,9 +97,5 @@ public void testFetchPartitionedTopicMetadataWithCacheRefresh() throws Exception
         PartitionedTopicMetadata partitionedTopicMetadata3 =
                 pulsar.getBrokerService().fetchPartitionedTopicMetadataAsync(topicName2, true).get();
         assertEquals(partitionedTopicMetadata3.partitions, 3);
-
-        // cleanup.
-        admin.topics().deletePartitionedTopic(topicName2.toString());
```

Comment:
can we skip this topic cleanup?

Reply:
yes, each run of the test will create a new namespace so there's no need to do the cleanup in the first place.